python/tests/scripts/e1_kb_write.metta

```diff
@@ -1,5 +1,12 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; MeTTa allows to use a separate knowledge base (AtomSpace) &kb
```

Comment:
We don't use "AtomSpace" in MeTTa implementation. Maybe, it is better to call it Space here, although calling it AtomSpace in some other contexts is perfectly ok.